oligo_designer_toolsuite/pipelines/_oligo_seq_probe_designer.py

```diff
@@ -788,48 +784,44 @@ def filter_by_specificity(
         """
 
         def _get_alignment_method(
-            alignment_method,
-            sequence_type,
-            search_parameters,
-            hit_parameters,
-            filter_name,
+            alignment_method: str,
+            search_parameters: dict,
+            hit_parameters: dict,
+            filter_name: str,
+            dir_output: str,
         ):
             if alignment_method == "blastn":
                 return BlastNFilter(
-                    sequence_type=sequence_type,
                     search_parameters=search_parameters,
                     hit_parameters=hit_parameters,
-                    dir_output=self.dir_output,
                     filter_name=filter_name,
+                    dir_output=dir_output,
                 )
             elif alignment_method == "bowtie":
                 return BowtieFilter(
-                    sequence_type=sequence_type,
                     search_parameters=search_parameters,
-                    dir_output=self.dir_output,
                     filter_name=filter_name,
+                    dir_output=dir_output,
                 )
             else:
                 raise ValueError(f"The alignment method {alignment_method} is not supported.")
 
         ##### define reference database #####
-        reference_database = ReferenceDatabase(
+        reference_database_alignment = ReferenceDatabase(
             database_name=f"{self.subdir_db_reference}_sequences", dir_output=self.dir_output
         )
-        reference_database.load_database_from_file(
-            files=files_fasta_reference_database, file_type="fasta", database_overwrite=False
+        reference_database_alignment.load_database_from_file(
+            files=files_fasta_reference_database, file_type="fasta", database_overwrite=True
         )
 
         reference_database_variants = ReferenceDatabase(
             database_name=f"{self.subdir_db_reference}_variants", dir_output=self.dir_output
         )
         reference_database_variants.load_database_from_file(
-            files=files_vcf_reference_database, file_type="vcf", database_overwrite=False
+            files=files_vcf_reference_database, file_type="vcf", database_overwrite=True
         )
 
-        ##### specificity filters #####
-        # remove sequences that could cause read length biases because the first
-        # <target_probe_read_length_bias> bases of both sequences match
+        ##### define exact match filter #####
```

Comment:
does it make sense to remove the comments from the previous version that provided context to the steps?